guides/bug_report_templates/action_mailbox_main.rb

```diff
@@ -9,6 +9,7 @@
 
   gem "rails", github: "rails/rails", branch: "main"
   gem "sqlite3"
+  gem "net-smtp"
```

Comment:
@yahonda Should this be added to [`actionmailbox.gemspec`](https://github.com/rails/rails/blob/main/actionmailbox/actionmailbox.gemspec)?

Reply:
Thanks for the review. I've found actonmailbox.gemspec already has dependency on `mail` gem.

https://github.com/rails/rails/blob/3c165ad2e70bef4e72f9cea3078ea4931b708656/actionmailbox/actionmailbox.gemspec#L40

```
  s.add_dependency "mail", ">= 2.7.1"
```

Once the newer versions of `mail` gem which includes this pull request https://github.com/mikel/mail/pull/1439 , 
at least we do not need to add another dependency to `net/smtp` gem inside of actonmailbox.gemspec .

I prefer to adding dependency inside of each test file by using `require 'bundler/inline' ` like https://github.com/rails/rails/commit/48f2f1aa078 did for Gemfile. We can remove these dependency once newer verions of `mail` is released.

Reply:
If we can remove these dependency once newer verions of `mail` is released, how about adding the `mail` with head dependency with the note instead?

Reply:
ah https://github.com/mikel/mail/pull/1439 isn't merged in master yet, nvm.